### ISSUE TITLE
Boolean checkbox autosave

### ISSUE THREAD

**Issue by elGusto** [aldeed/meteor-autoform#837]
Hi @aldeed 

I noticed a bug with autosave and the boolean checkbox input type. It's value is hard coded to true in the template and that value is then used to evaluate change in the form in the 'change form' event listener. This causes the autosave to not work after the first change caused by the checkbox.

What would be the correct way to handle this ? I tried some stuff out but turned out unsuccessful...

To be clear:

```html
<template name="afCheckbox">
  <div><label><input type="checkbox" value="true" {{atts}} /> {{afFieldLabelText name=this.name}}</label></div>
</template>
``` 

```javascript
    // Some plugins, like jquery.inputmask, can cause infinite
    // loops by continually saying the field changed when it did not,
    // especially in an autosave situation. This is an attempt to
    // prevent that from happening.
    var keyVal = key + '___' + event.target.value;
    if (keyVal === lastKeyVal) {
      return;
    }
    lastKeyVal = keyVal;

**Comment by jerefrer** [comment]
+1 This is really blocking for me. One just cannot use checkboxes with autosave enabled.

Here's a little GIF showing the bug in action. You can see that the first time clicking the checkbox actually persisted the value, but the couple next times did not.

![autosave_checkbox_bug](https://cloud.githubusercontent.com/assets/140954/7571033/165c035a-f815-11e4-9d3d-22958d27b235.gif)

**Comment by freer4** [comment]
+1 - I've taken to using boolean-radios for the moment, but this is definitely a bug that I could use addressed.

**Comment by elGusto** [comment]
FIXED with AutoForm v5.3.0

**Comment by jerefrer** [comment]
@elGusto Thanks a bunch !
